extensions/amp-base-carousel/0.1/test-e2e/test-advance.js

```diff
@@ -20,8 +20,9 @@ const pageWidth = 500;
 const pageHeight = 800;
 
 describes.endtoend(
-  'amp-base-carousel:0.1 - advance',
+  'amp-base-carousel - advance',
   {
+    version: '0.1',
```

Comment:
To confirm, does the console output for failed tests include the component version? The original reason they were added to the test names was so we could identify which version to debug.

Reply:
Yes; `describes.endtoend` embeds the version # inside the test name when the param is present:
![image](https://user-images.githubusercontent.com/6694512/106796797-d5b47d80-6629-11eb-8a66-9839f865a0a8.png)

Reply:
Cool, thanks!